templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -201,7 +201,7 @@ parameters:
 - description: Container Image to use for Ansible Service Broker in format of imagename:tag
   displayname: Ansible Service Broker Image
   name: BROKER_IMAGE
-  value: ansibleplaybookbundle/ansible-service-broker:latest
+  value: ansibleplaybookbundle/origin-ansible-service-broker:latest
```

Comment:
hopefully, this doesn't have to be swapped out for ose- later. I think we're ok for now.